open-metadata-implementation/user-interfaces/presentation-server/nodejs/src/client/components/Dino/README.md

```diff
@@ -6,66 +6,76 @@ The Dino interface provides a way to visualize Egeria platforms, servers, the se
 
 
 ### Deployment
-The Dino interface runs under the Presentation Server component of Egeria, and it uses the Dino View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Server please refer to their readme information.
+The Dino interface runs under the Presentation Server component of Egeria, and it uses the Dino View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Services please refer to their readme information, located here [Presentation Server](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/user-interfaces/presentation-server/README.md) and [View Services](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/view-services/README.md).
```

Comment:
The presentation server should be documented in the admin guide and that file is where this text should point to.

Reply:
Thanks - fair comment. In the latest push to this PR I have refactored so that the Dino documentation now points to the Admin Guide. I have written the necessary section of Admin Guide and it is in PR #3840 which includes a section on configuring presentation-server and a major extension of the section on conifgring view-services.